practice-one/src/index.html

```diff
@@ -3,44 +3,80 @@
+        <div class="footer-content container">
+            <div class="footer-item">
+                <div class="item-top">
+                    <h4 class="item-top-title">contact us</h4>
+                    <address class="item-top-address">
+                        <p class="description">2300 W Arbutus Dr Citrus Springs, FL 34434, USA</p>
+                        <div>  
+                            <div>
+                                <a class="description" href="tel:+2357938917938">+235 793-891-7938</a>
+                            </div>
+                            <div>
+                                <a class="description" href="mailto:contact@hofmannui.net">contact@hofmannui.net</a>
+                            </div>
+                        </div>
+                    </address>
+                </div>
+                <div class="item-top">
+                    <h4 class="item-top-title">follow us</h4>
+                    <div class="item-top-icons">
+                        <a href="#">
+                            <img class="description" src="assets/images/icons/Pinterest.svg" alt="Pinterest"/>
+                        </a>
+                        <a href="#">
+                            <img class="description" src="assets/images/icons/twitter.svg" alt="Twitter"/>
+                        </a>
+                        <a href="#">
+                            <img class="description" src="assets/images/icons/facebook.svg" alt="Facebook"/>
+                        </a>
+                    </div>
+                </div>
+                <div class="item-top">
+                    <h4 class="item-top-title">about us</h4>
+                    <p class="description">SmartWeb was made to help you quickly create beautiful<br>layouts of responsive websites.</p>
+                </div>  
+            </div>
+
+            <hr class="footer-separator">
+
+            <div class="footer-item">
+                <div class="description">© 2023 SmartWeb</div>
+                <nav>
+                    <ul class="menu">
+                        <li>
+                            <a class="menu-item" href="#">home</a>                    
+                        </li>
+                        <li>
+                            <a class="menu-item" href="#">places</a>
+                        </li>
+                        <li>
+                            <a class="menu-item" href="#">products</a>
+                        </li>
+                        <li>
+                            <a class="menu-item" href="#">projects</a>
+                        </li>
+                        <li>
+                            <a class="menu-item" href="#">portfolio</a>
+                        </li>
+                        <li>
+                            <a class="menu-item" href="#">about us</a>
+                        </li>
+                    </ul>
+                </nav>
+                <a href="#">
```

Comment:
@HoangLiarPhuongLe Please use `href="javascript:void(0)"` instead of `href="#"` so that the link will have no associated action. Clicking on it will not cause the page to reload or redirect to another location.